### ISSUE TITLE
Naming: "source" vs. "field"

### ISSUE THREAD

**Issue by mgeier** [sfstoolbox/sfs-python#27]
Currently, we have the submodules `sfs.mono.source` and `sfs.time.source`.

This makes sense for now, but we could generalize it by calling it something with "field", because that's actually what the functions generate.

**Comment by hagenw** [comment]
They can also generate a single point.

**Comment by mgeier** [comment]
@hagenw That's a good point! In the future, time-domain sources may even be able to generate a time signal for a single point.

**Comment by fs446** [comment]
These function create acoustical transfer functions either in frequency domain or time domain from the source location (exception plane wave) to one or many receiver locations, right? Typical terms used in acoustics are ATF and AIR, acoustical transfer function, acoustical impulse response. Some of these functions then are even Green's functions.

**Comment by mgeier** [comment]
@fs446 Yes, but in the future, the time-domain functions might also be able to generate time signals for given locations in space. I don't know if that counts as AIR.

Either way, are you suggesting a different module name (or multiple modules)?
